storage/client_test.go

```diff
@@ -91,6 +93,30 @@ func createTestStoreWithEngine(t *testing.T, eng engine.Engine, clock *hlc.Clock
 	return store, stopper
 }
 
+// multiStoreSender multiplexes send requests to a group of localSenders. It
+// sends the request to each localSender of a multiTestContext in order,
+// stopping if the request succeeds or any error other than RangeKeyMismatch is
+// returned.
+type multiStoreSender struct {
+	*multiTestContext
+}
+
+// Send implements the client.Sender interface.
+func (mss *multiStoreSender) Send(ctx context.Context, call proto.Call) {
```

Comment:
This doesn't need a new type; you can just define `Send` on `multiTestContext` and use that as your `Sender` (I have something like this in one of my branches). Add a TODO to remove this once #2141 is merged (which will make `multiTestContext` use a `DistSender` instead).